### PR TITLE
Add a workflow to build boot iso from a comment

### PR DESCRIPTION
**Pull request by jkonecny12**
Add build-boot-iso workflow which can be requested by the `/boot-iso` comment on a PR by the write access repository members.

This is based on https://github.com/rhinstaller/anaconda/pull/3979 . Thanks @KKoukiou for doing the first step!

Successful run https://github.com/jkonecny12/anaconda/actions/runs/2094814715 .

### CODE REVIEW

**Review by rvykydal — APPROVED** [review]
Looks good to me.

**Comment by jkonecny12** [comment]
I just realized that we need more work here. We need to build image with boot option for webUI set to avoid booting the GTK version. Back to the table...

**Review by KKoukiou — APPROVED** [review]
Thanks a lot for the updates! I can't test it without having it merged. So let's get it in and see it in action :)

File: dockerfile/anaconda-iso-creator/lorax-build [hunk 1]
```diff
@@ -20,18 +20,18 @@ REPO_DIR=/tmp/anaconda-rpms/
...
 # The download.fedoraproject.org automatic redirector often selects download-ib01.f.o. for GitHub's cloud, which is too unreliable; use a mirror
 # The --volid argument can cause different network interface naming: https://github.com/rhinstaller/kickstart-tests/issues/448
-lorax -p Fedora -v $VERSION_ID -r $VERSION_ID \
+lorax -p Fedora -v "$VERSION_ID" -r "$VERSION_ID" \
       --volid Fedora-S-dvd-x86_64-rawh \
       -s http://dl.fedoraproject.org/pub/fedora/linux/development/rawhide/Everything/x86_64/os/ \
       -s file://$REPO_DIR/ \
-      $@ \
-      lorax ||  cp *.log /images/
+      "$@" \
```
Comment (VladimirSlavik) [inline dockerfile/anaconda-iso-creator/lorax-build:34]: This is one of the cases where I am not so sure about blindly following shellcheck. Hopefully I am wrong.

**Review by VladimirSlavik — COMMENTED** [review]
Thank you, this is a great help!

**Pull request merged**